### PR TITLE
ReAdds Dildos

### PR DESCRIPTION
**Pull request by AliiTea**
April Fools fun please.

### CODE REVIEW

**Comment by Spacedong** [comment]
yes cunt

**Comment by doobiedoo23** [comment]
-1

**Comment by Kokojos** [comment]
It was fun for a whille, but two years in a row? Cmon.

**Comment by SplergeBucket** [comment]
merged

**Comment by AliiTea** [comment]
Oh shit is it actually getting merged or is looping making me sad

**Comment by Zadeon** [comment]
No please.

**Comment by danhibiki** [comment]
die

**Comment by TDcoolguy300** [comment]
HE HAS BEEN SUMMONED

**Comment by Kokojos** [comment]
WE NEED MORE LABELS. CAN THIS GO IN THE POOL?

**Comment by AliiTea** [comment]
Yes it can.

**Comment by Nokington** [comment]
No

**Comment by AliiTea** [comment]
You sir, are no fun.

**Comment by myontasm** [comment]
NO!!!!!!!!!!!!!!!!!!!!!!!!!!!!

File: code/game/objects/items/dildo [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	icon = 'icons/obj/dicks.dmi'
+	icon_state = null
+	force = 5
+	w_class = 1
+	throwforce = 5
+	hitsound = 'sound/items/squishy.ogg'
+	attack_verb = list("slapped")
+
+/obj/item/dragon/suicide_act(mob/user)
+	user.visible_message(pick("<span class='suicide'>[user] is shoving [src.name] down \his throat! It looks like they're trying to commit suicide.</span>"))
```
Comment (Carbonhell) [inline code/game/objects/items/dildo:15]: Useless pick()

File: code/game/objects/items/dildo [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	return(BRUTELOSS)
+
+/obj/item/dragon/sea
+	name = "Sea Dragon Dildo"
+	desc = "A Sea Dragon Dildo, Why this is on a SpaceStation we will never know..."
+	icon_state = "seadragon"
+	force = 5
```
Comment (Carbonhell) [inline code/game/objects/items/dildo:22]: No need to redefine force and throwforce since parent already has them to the same value

**Comment by Carbonhell** [comment]
Spoiled meme

**Comment by nalarmg** [comment]
Two heads have denied this.  Closing for now.

**Pull request closed without merging**